website/package.json

```diff
@@ -58,6 +58,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-github-btn": "^1.4.0",
+    "react-tweet": "3.2.1",
```

Comment:
Is this lib imported anywhere? looks like you embed the Metro tweet as an iframe in the end